py/torch_tensorrt/dynamo/test/test_dynamo_backend.py

```diff
@@ -31,6 +31,8 @@ def test_resnet18(ir):
         "enabled_precisions": {torch.float},
         "ir": ir,
         "pass_through_build_failures": True,
+        "optimization_level": 1,
+        "min_block_size": 8,
```

Comment:
Decreased optimization level (from 3 to 1) and increased block size (from 5 to 8) to improve compilation times and reduce segmentation in models, pending upcoming converter implementations.